resources/views/layouts/default/flights/simbrief_briefing.blade.php

```diff
@@ -284,6 +292,31 @@ class="btn btn-outline-primary pull-right"
       @endif
     </div>
   </div>
+
+  {{-- SimBrief Edit Modal --}}
+  @if(!empty($simbrief->xml->params->static_id))
+    <div class="modal fade" id="OFP_Edit" data-backdrop="static" data-keyboard="false" tabindex="-1" aria-labelledby="staticBackdropLabel" aria-hidden="true">
+      <div class="modal-dialog" style="max-width: 1020px;">
+        <div class="modal-content p-0" style="border-radius: 5px;">
+          <div class="modal-header p-1">
+            <h5 class="modal-title m-1 p-0">SimBrief</h5>
+            <span class="close"><i class="fas fa-times-circle" data-dismiss="modal" aria-label="Close" aria-hidden="true"></i></span>
+          </div>
+          <div class="modal-body p-0">
+            <iframe src="https://www.simbrief.com/system/dispatch.php?editflight=last&static_id={{ $simbrief->xml->params->static_id }}" style="width: 100%; height: 80vh;" frameBorder="0" title="SimBrief"></iframe>
```

Comment:
Is the iframe the right way to do this? Just makes sure against their API docs - I seem to remember something about the iframe being discouraged. Also, make the simbrief URL a config option and have the full URL be passed in via controller

Reply:
There is not other way to edit a generated flight plan, and this url is directly mentioned in the API docs for editing them.